drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -0,0 +1,667 @@
+	depends on NRF7000
+	help
+	  Select this option to enable scan only mode of the nRF70 driver
+
+config NRF70_RADIO_TEST
+	bool "Radio test mode of the nRF70 driver"
+
+config NRF70_SYSTEM_WITH_RAW_MODES
+	bool "nRF70 system mode with raw modes"
+	depends on NRF7002 || NRF7001
+	select WIFI_NM_WPA_SUPPLICANT
+	help
+	  Select this option to enable system mode of the nRF70 driver with raw modes
+
+endchoice
+
+config NET_L2_ETHERNET
+	default y if !NRF70_RADIO_TEST
+
+config HEAP_MEM_POOL_ADD_SIZE_NRF70
+	# Use a maximum that works for all samples/apps, each sample/app can override
+	# this value if needed by using CONFIG_HEAP_MEM_POOL_IGNORE_MIN
+	def_int 230000
+
+if NRF70_SYSTEM_MODE || NRF70_SYSTEM_WITH_RAW_MODES
+config NRF70_STA_MODE
+	bool "nRF70 STA mode"
+	default y
+	help
+		Select this option to enable STA mode of the nRF70 driver
+
+config NRF70_AP_MODE
+	bool "Access point mode"
+	depends on WIFI_NM_WPA_SUPPLICANT_AP
+
+config NRF70_P2P_MODE
+	bool "P2P support in driver"
+endif # NRF70_SYSTEM_MODE || NRF70_SYSTEM_WITH_RAW_MODES
+
+config NRF70_RAW_DATA_TX
+	bool "RAW TX data path in the driver"
+	select EXPERIMENTAL
+
+config NRF70_RAW_DATA_RX
+	bool "RAW RX sniffer operation in the driver"
+	select EXPERIMENTAL
+
+config NRF70_PROMISC_DATA_RX
+	bool "promiscuous RX sniffer operation in the driver"
+	select WIFI_NM_WPA_SUPPLICANT
+	select EXPERIMENTAL
+	select NET_PROMISCUOUS_MODE
+
+config NRF70_DATA_TX
+	bool "TX data path in the driver"
+	default y if NRF70_SYSTEM_MODE || NRF70_SYSTEM_WITH_RAW_MODES
+
+config NRF_WIFI_IF_AUTO_START
+	bool "Wi-Fi interface auto start on boot"
+	default y
+
+config NRF_WIFI_PATCHES_BUILTIN
+	bool "Store nRF70 FW patches as part of the driver"
+	default y
+	help
+	  Select this option to store nRF70 FW patches as part of the driver.
+	  This option impacts the code memory footprint of the driver.
+
+config CUSTOM_LINKER_SCRIPT
+	string "Custom linker script for nRF70 FW patches"
+	default "${ZEPHYR_BASE}/../nrf/drivers/wifi/nrf70/rpu_fw_patches.ld"
+
+config NRF_WIFI_LOW_POWER
+	bool "low power mode in nRF Wi-Fi chipsets"
+	default y
+
+config NRF70_TCP_IP_CHECKSUM_OFFLOAD
+	bool "TCP/IP checksum offload"
+	default y
+
+config NRF70_REG_DOMAIN
+	string "The ISO/IEC alpha2 country code for the country in which this device is currently operating. Default 00 (World regulatory)"
+	# 00 is used for World regulatory
+	default "00"
+
+# Making calls to RPU from net_mgmt callbacks.
+#
+# If WPA supplicant is enabled, then don't override as it has higher
+# stack requirements.
+config NET_MGMT_EVENT_STACK_SIZE
+	default 2048 if !WIFI_NM_WPA_SUPPLICANT
+
+config NRF70_LOG_VERBOSE
+	bool "Maintains the verbosity of information in logs"
+	default y
+
+module = WIFI_NRF70
+module-dep = LOG
+module-str = Log level for Wi-Fi nRF70 driver
+module-help = Sets log level for Wi-Fi nRF70 driver
+source "subsys/logging/Kconfig.template.log_config"
+
+config WIFI_NRF70_LOG_LEVEL
+	# Enable error by default
```

Comment:
Place comment in help section

Reply:
This symbol is an override, so, will help work? AFAIK, help and type are only for the original definition, no?

Reply:
right, ignore this comment :)